core/trino-main/src/test/java/io/trino/execution/TestNodeScheduler.java

```diff
@@ -311,7 +311,7 @@ public void testBasicAssignment()
         Multimap<InternalNode, Split> assignments = nodeSelector.computeAssignments(splits, ImmutableList.copyOf(taskMap.values())).getAssignments();
         assertThat(assignments.entries().size()).isEqualTo(assignments.size());
         for (InternalNode node : activeCatalogNodes) {
-            assertThat(assignments.keySet().contains(node)).isTrue();
+            assertThat(assignments.keySet()).contains(node);
```

Comment:
Can be further reduced to

```suggestion
            assertThat(assignments).containsKey(node);
```

Reply:
not possible since assignments is Multimap, not Map

Reply:
Yeah, I figure that after the fact